kong/router/atc.lua

```diff
@@ -2,6 +2,7 @@ local _M = {}
 local _MT = { __index = _M, }
 
 
+local tb_nkeys = require("table.nkeys")
```

Comment:
Moving this line below `tb_new` is better.

Reply:
Done